doc/api/debugger.md

```diff
@@ -182,4 +182,16 @@ process or via URI reference to the listening debugger:
 * `node debug <URI>` - Connects to the process via the URI such as
 localhost:5858
 
+## V8 Inspector Integration for Node.js
+
+V8 Inspector integration allows attaching Chrome DevTools to Node.js
+instances for debugging and profiling.
+
+V8 Inspector can be enabled by passing `--inspect` flag when starting Node.js
```

Comment:
If we are hitting path length issues we may want to try and do a test build
in our release ci to make sure there is no weirdness on Windows. I doubt it
will be a problem.

On my phone right now, but I'll kick off a build in the next little bit.